nuxeo-management-rest-api/src/main/java/org/nuxeo/rest/management/probes/ProbeInfoWriter.java

```diff
@@ -0,0 +1,95 @@
+/*
+ * (C) Copyright 2019 Nuxeo (http://nuxeo.com/) and others.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * Contributors:
+ *     Nour Al Kotob
+ */
+
+package org.nuxeo.rest.management.probes;
+
+import static org.nuxeo.ecm.core.io.registry.reflect.Instantiations.SINGLETON;
+import static org.nuxeo.ecm.core.io.registry.reflect.Priorities.REFERENCE;
+
+import java.io.IOException;
+
+import org.nuxeo.ecm.core.io.marshallers.json.AbstractJsonWriter;
+import org.nuxeo.ecm.core.io.registry.reflect.Setup;
+import org.nuxeo.ecm.core.management.api.ProbeInfo;
+
+import com.fasterxml.jackson.core.JsonGenerator;
+
+/**
+ * see {@link AbstractJsonWriter}
+ *
+ * @since 11.1
+ */
+@Setup(mode = SINGLETON, priority = REFERENCE)
+public class ProbeInfoWriter extends AbstractJsonWriter<ProbeInfo> {
+
+    public static final String WARNINGS = "warnings";
+
+    public static final String WARNING = "warning";
+
+    public static final String NAME = "name";
+
+    public static final String VERSION = "version";
```

Comment:
maybe i lmiss something, i checkout your branh and theses fields are unused.

Reply:
You didn't, actually my IDE doesn't highlight them but they are indeed useless now.